lib/shippo/api/request.rb

```diff
@@ -91,7 +91,14 @@ def execute
       def shippo_phone_home
         payload     = {}
```

Comment:
Just changing this to `payload = nil` is a smaller change (but is effectively the same).